spec/dummy/db/migrate/20160126082137_create_synced_timestamps.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSyncedTimestamps < ActiveRecord::Migration
+  def change
+    create_table :synced_timestamps do |t|
+      t.belongs_to :parent_scope, polymorphic: true, null: false
+      t.datetime :synced_at, null: false
+    end
+
+    add_index :synced_timestamps, [:parent_scope_id, :parent_scope_type, :synced_at], name: 'synced_timestamps_max_index', order: { synced_at: 'DESC' }
```

Comment:
might be due to sqlight, not even sure it supports such ordered indexes, but i think i got similar issue when trying ordered indexes in bsa-booking as well